### DISCUSSION TITLE
Adding WebToolkit aka WT as a dependency in a Meson project

### DISCUSSION THREAD

**Discussion by ghost** [mesonbuild/meson#12703] [Q&A]
I'm trying to use WebToolkit (installed via Msys2) as a dependency to my executable:
```
project(
    'Hello, Wt!',
    'cpp',
    version: '1.0.0'
)

wt_dep = dependency(
    'wt'
)

executable(
    'hello_wt',
    'main.cpp',
    dependencies: wt_dep
)
```
but neither CMake nor pkgconfig are able to find it.
Does anyone know how I can do it? Using Msys2 is not mandatory, I can also compile the source code of the library if it's the only way.

**Comment by eli-schwartz** [comment]
From the log file shared on IRC:
```
  -- Module search paths:    ['C:/Program Files', 'C:/Program Files (x86)', 'C:/Program Files/CMake']
  -- CMake root:             C:/Program Files/CMake/share/cmake-3.27
  -- CMake architectures:    []
  -- CMake lib search paths: ['lib', 'lib32', 'lib64', 'libx32', 'share']
Preliminary CMake check failed. Aborting.
```

This is failing because cmake isn't configured to check inside the msys2 directories.
